docs/source/en/using-diffusers/using_safetensors.mdx

```diff
@@ -1,16 +1,61 @@
-# What is safetensors ? 
+# Load safetensors
+
+[safetensors](https://github.com/huggingface/safetensors) is a safe and fast file format for storing and loading tensors. Typically, PyTorch model weights are saved or *pickled* into a `.bin` file with Python's [`pickle`](https://docs.python.org/3/library/pickle.html) utility. However, `pickle` is not secure and pickled files may contain malicious code that can be executed. safetensors is a secure alternative to `pickle`, making it ideal for sharing model weights.
+
+This guide will help you load `.safetensor` weights, and how to convert model weights stored in other formats to `.safetensor`. Before you start, make sure you have safetensors installed:
+
+```bash
+!pip install safetensors
+```
+
+If you look at the [`runwayml/stable-diffusion-v1-5`](https://huggingface.co/runwayml/stable-diffusion-v1-5/tree/main) repository, you'll see weights inside the `text_encoder`, `unet` and `vae` subfolders are stored in the `.safetensors` format. Load these `.safetensors` weights by setting `use_safetensors=True` and passing the model repository id to the [`~DiffusionPipeline.from_pretrained`] method:
+
+```py
+from diffusers import DiffusionPipeline
+
+pipeline = DiffusionPipeline.from_pretrained("runwayml/stable-diffusion-v1-5", use_safetensors=True)
```

Comment:
I thought the `.safetensor` files are loaded by default, but when I looked at the docs [here](https://huggingface.co/docs/diffusers/en/api/diffusion_pipeline#diffusers.DiffusionPipeline.from_pretrained.use_safetensors) that seems a bit contradictory. The docstring implies you need to set `use_safetensors=True` to load the `.safetensor` weights but the default is `None`.

Reply:
If `safetensors` is installed in your environment, then the safetensor weights are loaded by default. Otherwise, for more explicit control, `use_safetensors=True` is the way to go. a

Cc: @patrickvonplaten

Reply:
`.safetensor files`is loaded by default and but you need `use_safetensors` argument for 2 reasons (based on my understanding) :
1. if we set `use_safetensors=True` but it's not installed, we will get an error message
2. if we have `safetensors` installed but we don't want to load `.safetensors` file, we can set it to be `False`

Reply:
+1 to what @yiyixuxu and @sayakpaul say here